### PR TITLE
Character table fix, AP excess fix

### PR DESCRIPTION
**Pull request by ghost**
[redacted]

### CODE REVIEW

File: src/main/java/game/user/User.java [hunk 1]
```diff
@@ -351,9 +351,11 @@ public int incEXP(int inc, boolean onlyFull) {
                     }
                     Pointer<Boolean> reachMaxLev = new Pointer<>(false);
                     if (ExpAccessor.tryProcessLevelUp(character, basicStat, inc, reachMaxLev)) {
+						if (character.getCharacterStat().getJob() == 0) {
```
Comment (Arnuh) [inline src/main/java/game/user/User.java:354]: Your code does the same thing as what's on the master branch but with more code. I tested before pushing to master and it works fine so I'm assuming you somehow messed up applying the changes on master when testing.

**Comment by Arnuh** [comment]
While the swap of `characters` to `characters` would help anyone who doesn't fully understand SQL and its keywords I'm not sure if I would accept changing it just because of that reason.

**Comment by bcolby45** [comment]
It is extremely bad practice to use reserved words in any matter other than what they were intended for. As for the User.java changes, you did not test what you pushed to the master branch. The code you wrote gives you 5 AP per level up regardless, THEN checks to see if the user's job isn't 0 and if that is true, gives you 5 MORE AP plus 3 SP. I corrected this mistake.

**Comment by Arnuh** [comment]
You corrected no mistake because what you're saying isn't correct. If it's actually giving double AP then a bug exists somewhere else. https://github.com/EricSoftTM/OrionAlpha/blob/master/src/main/java/game/user/User.java#L354

And according to the [Mysql Docs](https://dev.mysql.com/doc/refman/8.0/en/keywords.html) quoting the text is valid and it says nothing against trying to avoid doing it. The best practice to quote or to just avoid keywords is a per-person opinion and both can create issues depending on what SQL you even use.

**Comment by ghost** [comment]
I did actually make a mistake when applying the code from the master branch. Also, my experience is mostly with Oracle SQL, so I did not know that using reserved words in the way you did wasn't completely advised against.

**Pull request closed without merging**